stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1318,3 +1322,18 @@
         u0
     )
 )
+
+;; What is the given reward cycle's stackers' aggregate public key?
+;; *New in Stacks 3.0*
+(define-read-only (get-aggregate-public-key (reward-cycle uint))
+    (map-get? aggregate-public-keys reward-cycle)
+)
+
+;; Set the aggregate public key to the provided value
```

Comment:
You don't need to remove this _per se_; you should instead just make it a private function, and have the node itself call it at the start of each PoX-4 reward cycle. You don't have to do this in this PR, but it does need to happen prior to Neon because this function should not be publicly reachable.

Reply:
Please open a Neon-tagged issue for the above :pray:

Reply:
I have created and also adding as comment in code: https://github.com/stacks-network/stacks-core/issues/4101